contents/tutorials/broken-link-checker.md

```diff
@@ -0,0 +1,132 @@
+---
+title: How to create a broken link (404) checker
+date: 2023-05-31
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-12.png
+tags: ['actions', 'configuration']
+---
+Broken links and 404s are frustrating for users. Without a way to check for them, you might not realize they exist and can’t fix them. This tutorial shows you how to create a broken link checker that sends a notification in Slack when a user visits a page that doesn’t exist.
+
+## Creating a Next.js app and adding PostHog
```

Comment:
I was surprised by seeing "next.js" here, since the title or intro dont mention it